build.sbt

```
@@ -4,25 +4,33 @@ lazy val scala_212 = "2.12.13"
 lazy val scala_213 = "2.13.4"
 
 lazy val V = new {
-  val avro4s        = "4.0.4"
-  val cats          = "2.4.2"
-  val `cats-effect` = "2.3.3"
-  val ciris         = "1.2.1"
-  val confluent     = "6.1.0"
-  val doobie        = "0.10.0"
-  val kafka         = "2.7.0"
-  val logback       = "1.2.3"
-  val `log-effect`  = "0.14.1"
-  val postgres      = "42.2.19"
-  val refined       = "0.9.21"
-  val scalacheck    = "1.15.3"
-  val scalatest     = "3.2.6"
-  val silencer      = "1.7.3"
-  val zio           = "1.0.4-2"
-  val `zio-s3`      = "0.3.0"
-  val `zio-interop` = "2.3.1.0"
-  val `zio-kafka`   = "0.14.0"
-  val sttp          = "3.1.7"
+  val avro4s       = "4.0.4"
+  val cats         = "2.4.2"
+  val ciris        = "1.2.1"
+  val confluent    = "6.1.0"
+  val doobie       = "0.10.0"
+  val kafka        = "2.7.0"
+  val logback      = "1.2.3"
+  val `log-effect` = "0.14.1"
+  val postgres     = "42.2.19"
+  val refined      = "0.9.21"
+
+  val scalacheck = "1.15.3"
+  val scalatest  = "3.2.6"
+  val silencer   = "1.7.3"
+  val sttp       = "3.1.7"
+
+  val zio         = "1.0.4-2"
+  val `zio-s3`    = "0.3.0"
+  val `zio-kafka` = "0.14.0"
+
+  val http4s = "1.0.0-M10" // last compatible with CE 2.3
+
+  private val `cats-effect-version` = "2.3"
+  val `cats-effect`                 = s"${`cats-effect-version`}.3"
+  val `zio-interop`                 = s"${`cats-effect-version`}.1.0"
+
+  val circeVersion = "0.12.3"
```

any reason why not
```suggestion
  val circeVersion = "0.13.0"
```
?